### PR TITLE
Changes for RHDEVDOCS-4829 Doc: ODC Usability & Customer Happiness Epic

### PR DESCRIPTION
**Pull request by shreyasiddhartha**
Change type: Doc: ODC Usability & Customer Happiness Epic
Doc JIRA: https://issues.redhat.com/browse/RHDEVDOCS-4829
Dev JIRA: https://issues.redhat.com/browse/ODC-7108
Fix Version: Openshift 4.13 Freeze

Note: The "Pods Receiving Traffic" feature will go as Release Notes in 4.13 which will be done through this JIRA: https://issues.redhat.com/browse/RHDEVDOCS-4867 so  the changes aren't included in this PR.

Doc Preview: https://55909--docspreview.netlify.app/openshift-enterprise/latest/applications/working_with_helm_charts/understanding-helm.html

SME Review: @debsmita1 
QE Review: @sanketpathak 
Peer Review: @rolfedh

### CODE REVIEW

File: applications/working_with_helm_charts/configuring-custom-helm-chart-repositories.adoc [hunk 1]
```diff
@@ -7,7 +7,7 @@ include::_attributes/common-attributes.adoc[]
 toc::[]
 
 [role="_abstract"]
-You can install Helm charts on an {product-title} cluster using the following methods:
+You can create Helm charts on an {product-title} cluster using the following methods:
```
Comment (debsmita1) [inline applications/working_with_helm_charts/configuring-custom-helm-chart-repositories.adoc:10]:
```suggestion
You can create Helm releases on an {product-title} cluster using the following methods:
```

File: applications/creating_applications/odc-creating-applications-using-developer-perspective.adoc [hunk 2]
```diff
@@ -88,4 +88,4 @@ include::modules/odc-exporting-an-application-to-another-project-or-cluster.adoc
...
 * For more information about Knative autoscaling settings for {ServerlessProductName}, see xref:../../serverless/develop/serverless-autoscaling-developer.adoc#serverless-autoscaling-developer[Autoscaling].
 * For more information about adding a new user to a project, see xref:../projects/working-with-projects.adoc#odc-providing-project-permissions-using-developer-perspective_projects[Working with projects].
-* For more information about creating a Helm Chart repository, see xref:../working_with_helm_charts/configuring-custom-helm-chart-repositories.adoc#odc-installing-helm-charts-using-developer-perspective_configuring-custom-helm-chart-repositories[Creating Helm Chart repositories].
+* For more information about creating a Helm Chart repository, see xref:../working_with_helm_charts/configuring-custom-helm-chart-repositories.adoc#odc-creating-helm-charts-using-developer-perspective_configuring-custom-helm-chart-repositories[Creating Helm Chart repositories].
```
Comment (debsmita1) [inline applications/creating_applications/odc-creating-applications-using-developer-perspective.adoc:91]:
```suggestion
* For more information about creating a Helm Chart repository, see xref:../working_with_helm_charts/configuring-custom-helm-chart-repositories.adoc#odc-creating-helm-releases-using-developer-perspective_configuring-custom-helm-chart-repositories[Creating Helm Chart repositories].
```

File: applications/working_with_helm_charts/configuring-custom-helm-chart-repositories.adoc [hunk 3]
```diff
@@ -28,7 +28,7 @@ In the *Developer* perspective of the web console, you can use the *Helm* page t
 
 include::modules/helm-installing-a-helm-chart-on-an-openshift-cluster.adoc[leveloffset=+1]
 
-include::modules/odc-installing-helm-charts-using-developer-perspective.adoc[leveloffset=+1]
+include::modules/odc-creating-helm-charts-using-developer-perspective.adoc[leveloffset=+1]
```
Comment (debsmita1) [inline applications/working_with_helm_charts/configuring-custom-helm-chart-repositories.adoc:31]:
```suggestion
include::modules/odc-creating-helm-releases-using-developer-perspective.adoc[leveloffset=+1]
```

File: modules/odc-creating-helm-charts-using-developer-perspective.adoc [hunk 4]
```diff
@@ -1,8 +1,8 @@
 :_content-type: PROCEDURE
-[id="odc-installing-helm-charts-using-developer-perspective_{context}"]
-= Installing Helm charts using the Developer perspective
+[id="odc-creating-helm-charts-using-developer-perspective_{context}"]
```
Comment (debsmita1) [inline modules/odc-creating-helm-charts-using-developer-perspective.adoc:2]:
```suggestion
[id="odc-creating-helm-releases-using-developer-perspective_{context}"]
```

File: modules/odc-creating-helm-charts-using-developer-perspective.adoc [hunk 5]
```diff
@@ -1,8 +1,8 @@
...
+= Creating Helm charts using the Developer perspective
```
Comment (debsmita1) [inline modules/odc-creating-helm-charts-using-developer-perspective.adoc:3]:
```suggestion
= Creating Helm releases using the Developer perspective
```

File: modules/odc-creating-helm-charts-using-developer-perspective.adoc [hunk 6]
```diff
@@ -1,8 +1,8 @@
...
 
-You can use either the *Developer* perspective in the web console or the CLI to select and install a chart from the Helm charts listed in the *Developer Catalog*. You can create Helm releases by installing Helm charts and see them in the *Developer* perspective of the web console.
+You can use either the *Developer* perspective in the web console or the CLI to select and create a chart from the Helm charts listed in the *Developer Catalog*. You can create Helm releases by creating Helm charts and see them in the *Developer* perspective of the web console.
```
Comment (debsmita1) [inline modules/odc-creating-helm-charts-using-developer-perspective.adoc:5]:
```suggestion
You can use either the *Developer* perspective in the web console or the CLI to select and create a release from the Helm charts listed in the *Developer Catalog*. You can create Helm releases by installing Helm charts and see them in the *Developer* perspective of the web console.
```

File: modules/odc-creating-helm-charts-using-developer-perspective.adoc [hunk 7]
```diff
@@ -27,8 +27,8 @@ image::odc_helm_chart_devcatalog.png[]
 Where available, you can switch between the *YAML View* and *Form View*. The data is persisted when switching between the views.
 ====
 +
-.. Click *Install* to create a Helm release. You will be redirected to the *Topology* view where the release is displayed. If the Helm chart has release notes, the chart is pre-selected and the right panel displays the release notes for that release.
+.. Click *Create* to create a Helm release. You will be redirected to the *Topology* view where the release is displayed. If the Helm chart has release notes, the chart is pre-selected and the right panel displays the release notes for that release.
```
Comment (debsmita1) [inline modules/odc-creating-helm-charts-using-developer-perspective.adoc:30]: If a Helm Chart creates workloads when installed, then the user is redirected to Topology, else is navigated to the Helm release details page.
Workloads are : DaemonSet, CronJob, Pod, Deployment, DeploymentConfig

File: modules/odc-deleting-helm-release.adoc [hunk 8]
```diff
@@ -1,7 +1,7 @@
 :_content-type: PROCEDURE
-[id="odc-uninstalling-helm-release_{context}"]
+[id="odc-deleting-helm-release_{context}"]
 = Uninstalling a Helm release
```
Comment (debsmita1) [inline modules/odc-deleting-helm-release.adoc:3]:
```suggestion
= Deleting a Helm release
```

**Review by debsmita1 — CHANGES_REQUESTED** [review]

File: modules/odc-creating-helm-charts-using-developer-perspective.adoc (hunk 7 above)
Comment (shreyasiddhartha) [reply modules/odc-creating-helm-charts-using-developer-perspective.adoc:30]: @debsmita1 thank you for the review. I have updated all the suggestions by you. Just want to clarify whether this is an addition to the above step mentioned step. If yes, please let me know.

**Comment by sanketpathak** [comment]
Thanks @shreyasiddhartha 
LGTM

File: applications/working_with_helm_charts/understanding-helm.adoc [hunk 9]
```diff
@@ -12,9 +12,9 @@ Helm is a software package manager that simplifies deployment of applications an
 Helm uses a packaging format called _charts_.
 A Helm chart is a collection of files that describes the {product-title} resources.
 
-A running instance of the chart in a cluster is called a _release_. A new release is created every time a chart is installed on the cluster.
+A running instance of the chart in a cluster is called a _release_. A new release is created every time a chart is created on the cluster.
```
Comment (rolfedh) [inline applications/working_with_helm_charts/understanding-helm.adoc:15]: For minimalism, maybe:
```suggestion
Creating a chart in a cluster creates a running instance of the chart known as a _release_.
```

File: modules/odc-creating-helm-releases-using-developer-perspective.adoc [hunk 10]
```diff
@@ -27,8 +27,8 @@ image::odc_helm_chart_devcatalog.png[]
 Where available, you can switch between the *YAML View* and *Form View*. The data is persisted when switching between the views.
 ====
 +
-.. Click *Install* to create a Helm release. You will be redirected to the *Topology* view where the release is displayed. If the Helm chart has release notes, the chart is pre-selected and the right panel displays the release notes for that release.
+.. Click *Create* to create a Helm release. You will be redirected to the *Topology* view where the release is displayed. If the Helm chart has release notes, the chart is pre-selected and the right panel displays the release notes for that release. If a Helm chart creates workloads when installed, then the user is redirected to *Topology*, or else is navigated to the *Helm release details* page. The workloads are DaemonSet, CronJob, Pod, Deployment, and DeploymentConfig.
```
Comment (rolfedh) [inline modules/odc-creating-helm-releases-using-developer-perspective.adoc:30]:
```suggestion
.. Click *Create* to create a Helm release. The <console?> displays the new release in the *Topology* view. 
+
If a Helm chart has release notes, the <console?> displays them. 
+
If a Helm chart creates workloads, the <console?> displays them on the *Topology* or *Helm release details* page. The workloads are `DaemonSet`, `CronJob`, `Pod`, `Deployment`, and `DeploymentConfig`.
```
- For clarity, avoid switching between different forms of address. For example, the content switches between addressing the user as "you" and "the user". Rather, use the dominant form of address consistently. 
- Use present tense; avoid future tense ("will").
- Evaluate removing/minimizing descriptions of the outcome of the step. For example, here, consider moving "The workloads are `DaemonSet`, `CronJob`, `Pod`, `Deployment`, and `DeploymentConfig`." to the topic overview or related a concept topic. The user might get this information from the screen when it happens. (However, if this information is essential for the user to see at that moment, for example, if it affects the decisions and actions, you can keep it there.)

**Review by rolfedh — APPROVED** [review]
LGTM. Nice work. I offered some style suggestions/coaching. HTH.

File: applications/working_with_helm_charts/understanding-helm.adoc (hunk 9 above)
Comment (shreyasiddhartha) [reply applications/working_with_helm_charts/understanding-helm.adoc:15]: Updated, thanks!

File: modules/odc-creating-helm-releases-using-developer-perspective.adoc (hunk 10 above)
Comment (shreyasiddhartha) [reply modules/odc-creating-helm-releases-using-developer-perspective.adoc:30]: Updated this. I have used "web console" in place of the console placeholders you have mentioned. Thank you for help.

**Pull request merged**